src/main/resources/db/migration/V1_5__referral_further_information.sql

```diff
@@ -0,0 +1,9 @@
+alter table referral
+    add column further_information text,
```

Comment:
_data question_
I'm still catching up so I'm a bit out of context, but that may be good. Is it worth adding context to these field names?

For example, by analysing the data in `further_information`, the processor will have to know if it's in the context of risk, the person or the desired actions.

Reply:
The question and hint are:

> Do you have further information for the ${this.serviceCategory.name} service provider? (optional)
> For example, relevant previous offences, previously completed programmes or further reasons for this referral

`further_information_for_service_provider`?